src/test/java/root/send/MessageTest.java

```diff
@@ -157,13 +156,14 @@ public void buildMessageWithDataMessage() {
         Assert.assertEquals(webhookEmbeds.size(), 1);
         
         WebhookEmbed webhookEmbed = webhookEmbeds.get(0);
-        
-        Assert.assertTrue(jdaMessage instanceof DataMessage);
+
         Assert.assertTrue(webhookMessage.isTTS());
         Assert.assertEquals(webhookMessage.getContent(), "myContent");
         Assert.assertEquals(webhookEmbed.getTitle().getText(), "myEmbed");
     }
-    
+
+    /* ReceivedMessage runs checkIntent when getContentRaw and getEmbeds are called
+     * This would require the test to have a JDA instance, as checkIntent needs it
```

Comment:
I think we can just remove the entire support for sending `Message`, since that isn't intended in JDA either.